terraform/terraform.tf

```diff
@@ -76,3 +76,23 @@ module "statuscake" {
   source = "./modules/statuscake"
   alerts = var.statuscake_alerts
 }
+
+# The following code imports the Azure module which just creates the temporary data storage account.
+# This is commented out because the CI pipeline service principal does not have sufficient permission
+# to create Azure resources at present but we wanted to document the temporary data storage account
+# in Terraform for when we migrate the service away from GOV.UK PaaS
+
+# module "azure" {
```

Comment:
Only this part needs to be commented out. The files in the module don't need to be

Reply:
I wasn't sure on this - but it's a lot nicer not having all the commented lines now!